libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/EHorizon.kt

```diff
@@ -0,0 +1,76 @@
+package com.mapbox.navigation.base.trip.model
+
+/**
+ * The EHorizon
+ *
+ * @param start
+ */
+data class EHorizon(
+    var start: Edge
+) {
```

Comment:
Will this class contain more parameters in the future? If that's the case, it cannot be a data class.

Reply:
@mskurydin @SiarheiFedartsou could you help here? Are we envisioning extending this / adding more data as part of `EHorizon` in the future?

Reply:
As I remember we expose here absolutely everything that we currently can extract from routing tiles, but nobody guarantee that we won't extend routings tiles in the future with some new info we want to expose in e-horizon.